client/src/elements/item-preview/item-preview.js

```diff
@@ -59,6 +48,21 @@ export class ItemPreview {
       }
     );
 
+    this.sizeOptions = [
```

Comment:
Do you think it would make sense to implement https://github.com/nzzdev/Q-editor/issues/4 together with this feature as it's touched now anyway?

Reply:
- I have implemented this in https://github.com/nzzdev/Q-editor/pull/119/commits/875a59761df172dddbde32013f6879e38f8ad780. And created a PR -> https://github.com/nzzdev/Q-server-nzz/pull/138 to add the previewSizes array to each targets editor config. 
- This allows every target to customize the preview width and label text

Reply:
👍